MAUI/MauiAppB2C/Platforms/Tizen/Main.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using Microsoft.Maui;
+using Microsoft.Maui.Hosting;
+
+namespace MauiB2C;

```

Comment:
I feel it's better to mention in the readme.md that Tizen and MacCatalyst are not supported.

Reply:
Done